dbms/src/TiDB/Schema/TiDB.cpp

```diff
@@ -172,6 +173,31 @@ Poco::JSON::Object::Ptr vectorIndexToJSON(const VectorIndexDefinitionPtr & vecto
     return vector_index_json;
 }
 
+InvertedIndexDefinitionPtr parseInvertedIndexFromJSON(IndexType index_type, const Poco::JSON::Object::Ptr & json)
+{
+    assert(json); // not nullptr
+
+    RUNTIME_CHECK(index_type == IndexType::INVERTED);
+    bool is_signed = json->getValue<bool>("is_signed");
```

Comment:
1. If the user change a column from signed int to unsigned int (or vice versa), does tidb reorg the data to new column_id?
2. If there are inverted index on the old column_id, after tidb reorg the data to new column_id, will there exist inverted index to the new column_id?

Reply:
1. Yes
2. If there are inverted index on the old column_id, modifying the column type that needs reorg is forbidden.